src/app/accessRequests/views/reviewServiceRequest.ejs

```diff
@@ -147,7 +147,13 @@
 
                     </fieldset>
                 </div>
-
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
```

Comment:
to make the text regular, <strong class="govuk-warning-text__text"> can be replaced with the following
<p class="govuk-warning-text__text govuk-!-font-weight-regular">